src/Sulu/Bundle/MediaBundle/Media/Manager/MediaManager.php

```diff
@@ -785,6 +787,26 @@ protected function setDataToMedia(Media $media, $data, $user)
                             $media->setType($type);
                         }
                         break;
+                    case 'categories':
+                        $categoryIds = [];
+                        $media->removeCategories();
+
+                        if (count($value)) {
```

Comment:
so we only need one select, if we combine it we need for every category one select